src/views/tutorial-view/index.tsx

```diff
@@ -182,7 +182,7 @@ function TutorialView({
 			children: (
 				<TutorialViewSidebarContent
 					items={collectionCtx.current.tutorials.map((t) =>
-						formatTutorialToMenuItem(t, collectionCtx.current.slug, currentPath)
+						formatTutorialToMenuItem(t, collectionCtx.current, currentPath)
```

Comment:
Updated args here as we need both collection `id` (for tutorial progress, which is specific to collection context) as well as `slug` (as previously, used to build the tutorial URL).